Formula/encfs.rb

```diff
@@ -23,11 +23,13 @@ class Encfs < Formula
 
   def install
     ENV.cxx11
-    ENV.append "CMAKE_INCLUDE_PATH", "/usr/local/include", ":"
     ENV.append "CMAKE_LIBRARY_PATH", "/usr/local/lib", ":"
 
+    args = std_cmake_args
+    args << "-DFUSE_INCLUDE_DIR=/usr/local"
```

Comment:
This is the purview of pkg-config. I'd like to see this fixed upstream not hacked in the formula. Please open an issue upstream.